files/headers.js

```diff
@@ -1,30 +1,35 @@
 import * as set_cookie_parser from 'set-cookie-parser';
 
+/**
+ * @typedef {import('@azure/functions').Cookie} Cookie
+ */
+
 /**
  * Splits 'set-cookie' headers into individual cookies
  * @param {Headers} headers
  * @returns {{
- *   headers: Record<string, string>,
- *   cookies: set_cookie_parser.Cookie[]
+ *   headers: Headers,
+ *   cookies: Cookie[]
  * }}
  */
 export function splitCookiesFromHeaders(headers) {
 	/** @type {Record<string, string>} */
 	const resHeaders = {};
 
-	/** @type {set_cookie_parser.Cookie[]} */
+	/** @type {Cookie[]} */
 	const resCookies = [];
 
 	headers.forEach((value, key) => {
 		if (key === 'set-cookie') {
 			const cookieStrings = set_cookie_parser.splitCookiesString(value);
+			// @ts-ignore
 			resCookies.push(...set_cookie_parser.parse(cookieStrings));
```

Comment:
Looks like this is due to a type mismatch on the `sameSite` property - are we sure the `sameSite` property returned by set_cookie_parser matches what Azure expects?

> Type 'string' is not assignable to type '"Strict" | "Lax" | "None"'